tools/util/include/cutlass/util/packed_stride.hpp

```diff
@@ -35,6 +35,7 @@
 #pragma once
 
 #include "cute/layout.hpp"
+#include "cutlass/conv/convolution.h"
```

Comment:
Is this header needed here?

Reply:
Surprisingly yes, otherwise `cutlass::conv` is not found

Reply:
Similarly also for the changes  in sm90_builder.inl

Reply:
Is this happening in upstream cutlass as well? If so we need to report the bug there

Reply:
It was an issue with how the header files were being included in the example, changing the order of the includes fixed this issue yesterday (https://github.com/codeplaysoftware/cutlass-fork/pull/122/commits/335f7f8cf207f8dfc463cb3fada34b3dba2fe8b7)

Reply:
> Is this happening in upstream cutlass as well

So I have removed the include in packed_stride.hpp (thanks @muhammad-tanvir-1211 )
But this is still required. 

The issue is that CUTLASS, for some reason relies on the files included, and even the include order of the app using it, which IMO is not a good solution.

I can try try building nvidia cutlass main example 49 and see what happens

Reply:
I figured out the issue, 
and was able to remove the extra include her in this commit [26ab12c](https://github.com/codeplaysoftware/cutlass-fork/pull/122/commits/26ab12c8a32219aab0c1069a514d98915aa74bd2)

@mehdi-goli @muhammad-tanvir-1211